redis/redis.go

```diff
@@ -2,14 +2,15 @@
 package redis
 
 import (
+	"time"
+
 	"github.com/garyburd/redigo/redis"
-	"github.com/gregjones/httpcache"
 )
 
 // cache is an implementation of httpcache.Cache that caches responses in a
 // redis server.
-type cache struct {
-	redis.Conn
+type Cache struct {
```

Comment:
Consistency; all others are public. It would give `*cache is not a type` trying to reference a pointer. Comment updated.